test/FieldTests/RandomiseMines.java

```diff
@@ -2,13 +2,14 @@
 
 import Field.*;
 import org.junit.jupiter.api.Test;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 public class RandomiseMines {
 
     @Test
     public void ifAFieldIsCreatedWithRandomMinePlacementTheFieldWillNotBeEmpty() {
-        Field field = new Field(5,5, new MockMineRandom(1, 0, 0));
+        Field field = new Field(new Size(5,5), new RandomMinePlacer(5, 5));
```

Comment:
Remove this test and write proper tests for random mine placer, but in a new branch.